pkg/lint/package.go

```diff
@@ -0,0 +1,297 @@
+package lint
+
+import (
+	"context"
+	"fmt"
+	"go/build"
+	"go/token"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"golang.org/x/tools/go/packages"
+
+	"github.com/golangci/golangci-lint/pkg/config"
+	"github.com/golangci/golangci-lint/pkg/exitcodes"
+	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis/load"
+	"github.com/golangci/golangci-lint/pkg/goutil"
+	"github.com/golangci/golangci-lint/pkg/lint/linter"
+	"github.com/golangci/golangci-lint/pkg/logutils"
+)
+
+// PackageLoader loads packages based on `golang.org/x/tools/go/packages.Load`.
```

Comment:
We can change to the following:
```suggestion
// PackageLoader loads packages based on [golang.org/x/tools/go/packages.Load].
```
to show active links in the doc:
<img width="635" alt="image" src="https://github.com/golangci/golangci-lint/assets/3228886/e088b6e9-a8e5-4b87-ac84-ef652f4ac63c">